publish/cli/make-bin-win32.sh

```diff
@@ -76,7 +76,14 @@ if [ -z "$LMS_NO_SIGN" ]; then
 
     # Try to sign the binary
     if [[ -n "${DIST_DIR}" ]] && [[ -n "${EXE_NAME}" ]]; then
-        smctl sign --keypair-alias="${WINDOWS_DIGICERT_KEYPAIR_ALIAS}" --input "${DIST_DIR}/${EXE_NAME}"
+        echo "Attempting to sign '${DIST_DIR}/${EXE_NAME}'..."
+        output=$(smctl sign --keypair-alias="${WINDOWS_DIGICERT_KEYPAIR_ALIAS}" --input "${DIST_DIR}/${EXE_NAME}" -v)
+        if [ $? -ne 0 ] || [[ ! $output == *"SUCCESSFUL"* ]]; then
```

Comment:
`smctl sign` was returning exit code 0 even on signing failures, which is the cause for this output comparison. Decided to go with a "need to see SUCCESSFUL" instead of "check for FAILED", to lower the possibility of a false positive